src/main/webapp/app/overview/course-overview.component.ts

```diff
@@ -61,12 +63,19 @@ export class CourseOverviewComponent implements OnInit, OnDestroy {
 
     loadCourse(refresh = false) {
         this.refreshingCourse = refresh;
-        this.courseService.findOneForDashboard(this.courseId).subscribe((res: HttpResponse<Course>) => {
-            this.courseCalculationService.updateCourse(res.body!);
-            this.course = this.courseCalculationService.getCourse(this.courseId);
-            this.adjustCourseDescription();
-            setTimeout(() => (this.refreshingCourse = false), 500); // ensure min animation duration
-        });
+        this.courseService.findOneForDashboard(this.courseId).subscribe(
+            (res: HttpResponse<Course>) => {
+                this.courseCalculationService.updateCourse(res.body!);
+                this.course = this.courseCalculationService.getCourse(this.courseId);
+                this.adjustCourseDescription();
+                setTimeout(() => (this.refreshingCourse = false), 500); // ensure min animation duration
+            },
+            (error: HttpErrorResponse) => {
+                const errorMessage = error.headers.get('X-artemisApp-message')!;
+                const jhiAlert = this.jhiAlertService.error(errorMessage);
+                jhiAlert.msg = errorMessage;
```

Comment:
I am still not quite sure what this line does as you passed the alert service the errorMessage already.

Reply:
I actually don't know either, but seeing this used in other places I thought it covers some edge case I have not thought of yet. Should I remove it?